docs/helpers/env_vars.yaml

```diff
@@ -1,3 +1,104 @@
+ACTIVITYLOG_SERVICE_ACCOUNT_ID:
+  name: OCIS_SERVICE_ACCOUNT_ID;ACTIVITYLOG_SERVICE_ACCOUNT_ID
+  defaultValue: ""
+  type: string
+  description: The ID of the service account the service should use. See the 'auth-service'
+    service description for more details.
+  introductionVersion: "5.0"
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+ACTIVITYLOG_STORE:
+  name: OCIS_PERSISTENT_STORE;ACTIVITYLOG_STORE
+  defaultValue: nats-js-kv
+  type: string
+  description: 'The type of the store. Supported values are: ''memory'', ''ocmem'',
+    ''etcd'', ''redis'', ''redis-sentinel'', ''nats-js'', ''noop''. See the text description
+    for details.'
+  introductionVersion: pre5.0
```

Comment:
This one is "pre5.0" - that seems weird - if the env var existed before 5.0, then why is it not already in the docs?